rdt-open-reader/src/main/java/io/ona/rdt/interactor/LoginInteractor.java

```diff
@@ -23,4 +27,12 @@ protected void scheduleJobsPeriodically() {
     public void scheduleJobsImmediately() {
         Utils.scheduleJobsImmediately();
     }
+
+    @Override
+    public void login(WeakReference<BaseLoginContract.View> view, String userName, char[] password) {
+        if (NetworkUtils.isNetworkAvailable()) {
+            CoreLibrary.getInstance().context().allSharedPreferences().saveForceRemoteLogin(true, userName);
```

Comment:
Why not set `https://github.com/OpenSRP/opensrp-client-core/blob/0cece3161b7aa92b865f2d8f895ac38272a314de/opensrp-app/src/main/java/org/smartregister/login/interactor/BaseLoginInteractor.java#L69` to true and allow the `super.login()` method to control the rest? As is this will attempt both remote and local login.